arangod/Aql/tokens.ll

```diff
@@ -126,7 +126,7 @@ class Parser;
   return T_DESC;
 }
 
-(?i:NOT[ \t\r\n]+IN) {
+(?i:NOT[ \t\r\n]+IN)/[ \t\r\n] {
```

Comment:
`/...` is Flex' version of `(?=...)` it seems and is called [trailing context](https://www.cs.virginia.edu/~cr4bd/flex-manual/Patterns.html#Patterns).

`(?i:NOT[ \t\r\n]+IN[ \t\r\n])` should equally work but consumes the trailing whitespace - which shouldn't make a difference because whitespace is ignored anyway. However, I wonder if consuming `\n` in tokens throws off the line/column count perhaps - I'm referring to this:
```
[\n] {
  yycolumn = 0;
}
```

Reply:
Will not work for `x NOT IN(`....

Reply:
Nor `x NOT IN[1,2,3]`.

Reply:
Good point, should instead check whether there is as non-variable name character